### PR TITLE
doc: how to start artipie with maven-proxy repository

### PR DESCRIPTION
**Pull request by dgarus**
Closes: #1059 

- written section `how to start artipie with maven-proxy repository`
- fixed maven-proxy example files

### CODE REVIEW

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@dgarus thanks, please, check my comments

**Review by Swizbiz — CHANGES_REQUESTED** [review]
please check my comments

File: .wiki/Home.md [hunk 1]
```diff
@@ -22,19 +22,85 @@ it's stored in storage as yaml files: for single-insance deployment it's
...
+      path: /var/artipie/data
+```
+- field `type` describes repository type, in our case it's `maven-proxy`.
+- field `url` points to a remote maven repository.
+- field `cache` describes storage to keep artifacts gotten from the remote maven repository.
+
+Detailed description for every supported repository type can be found [here](https://github.com/artipie/artipie/tree/master/examples).
+
+The file `my-maven.yaml` has to be placed on the path `/var/artipie/repo/my-maven.yaml`
+then Artipie service find it while startup and create repository with name `my-maven`.
```
[the remark this answers is not in the document]
Comment (dgarus) [reply .wiki/Home.md:68]: may be add `will`?

**Review by Swizbiz — APPROVED** [review]
thanks!

**Pull request merged**